src/simulate.js

```diff
@@ -53,7 +52,37 @@ module.exports = function (opts, simHostOpts) {
     });
 };
 
-function parseStartPage(projectRoot) {
+var closeServer = function() {
+    return server.server && server.server.close();
+};
+
+var launchBrowser = function(target, url) {
+    return cordovaServe.launchBrowser({ target: target, url: url });
+};
+
+var simulate = function(opts) {
+    
+    var target = opts.target || 'chrome';
+    var simHostUrl;
+
+    return launchServer(opts)
```

Comment:
We do need both. In VSCode, we only launch the server and the app-host via `launchServer()` and `launchBrowser`. Sim-host is launched differently.